conventional_precommit_linter/hook.py

```diff
@@ -26,11 +26,14 @@ def parse_args(argv: List[str]) -> argparse.Namespace:
         '--subject-min-length', type=int, default=20, help="Minimum length of the 'Summary' field in commit message"
     )
     parser.add_argument(
-        '--subject-max-length', type=int, default=50, help="Maximum length of the 'Summary' field in commit message"
+        '--subject-max-length', type=int, default=72, help="Maximum length of the 'Summary' field in commit message"
```

Comment:
`--sumary-max-length` would be a better name? (but it's a breaking change though...)

Reply:
@hfudev Probably yes, but I try to keep the same terminology as in the npm module `commitlint` (used by Danger).
If you don't mind, I'll leave it as it is for now.

Reply:
Sure. No problem!